.github/workflows/tests_without_binary.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04]
-        pl_backend: ["lightning_qubit", "lightning_kokkos"]
```

Comment:
Hey Vincent, why are you removing ``lightning_kokkos`` from the tests without binary?

Reply:
Yes, we need to install lightning_qubit to ensure package compatibility.
We shouldn't have any problems with the CI, could you please provide more context for that?
We want to keep testing lightning_kokkos without binaries.